app/models/feedback.rb

```diff
@@ -1,3 +1,4 @@
 class Feedback < ActiveRecord::Base
   validates :application, :username, :feedback, presence: true
+  enum status: [ :open, :in_progress, :closed ]
```

Comment:
Elsewhere in the Caseflow world, we've opted to not use `ActiveRecord::Enum` this way, since it's a bit tricky to change later on. Instead, we usually declare enums like:

```
  enum status: {
    open: 0,
    in_progress: 1,
    closed: 2,
  }
```
See https://bendyworks.com/blog/rails-enum-sharp-knife for an explanation of why.
And see `caseflow-efolder/app/models/download` for an example of this approach. Can we use that here?